File: bimestre1/aula_04-02/02-primo.c
#include <stdio.h>

int main() {
    int n = 0;
    int primo = 1;
    printf("Digite um número para verificar se é primo: ");
    scanf("%d",&n);

    for(int i = n-1; i > 1; i--){
        printf("%d%d\n",i,primo);
        if(n % i == 0) {
            primo = 0;
        }
    }

    if(primo == 1) {
        printf("O número %d é primo\n",n);
    } else {
        printf("O número %d não é primo\n",n);
    }
    
    return 0;
}

File: bimestre1/aula_04-16/02-primo.go
package main
import "fmt"

// Dado um inteiro positivo n, verificar se n é primo
func main() {
	var n int

	fmt.Print("Informe um número para verificar se é primo: ")
	fmt.Scanf("%d", &n)

	var primo bool = true

	for i := n-1; i > 1; i-- {
		if n % i == 0 {
			primo = false
		}
	}

	if primo {
		fmt.Printf("O número %d é primo\n", n);
	} else {
		fmt.Printf("O número %d não é primo\n", n);
	}
}
